incremental-font-transfer/src/patch_group.rs

```diff
@@ -726,6 +726,10 @@ mod tests {
         font_builder.build()
     }
 
+    const URL_TEMPLATE: &[u8] = &[
+        10, b'/', b'/', b'f', b'o', b'o', b'.', b'b', b'a', b'r', b'/', 128,
+    ];
```

Comment:
maybe deserves explaining the `10` and `128` bytes? but you can more easily express this as,

```suggestion
    const URL_TEMPLATE: &[u8] = b"\x0a//foo.bar/\x80";
```